### ISSUE TITLE
Explore Transformers

### ISSUE THREAD

**Issue by weka511** [weka511/nlp#15]

**Comment by weka511** [comment]
C:\Users\Weka\nlp\transformer.py:332: UserWarning: Implicit dimension choice for softmax has been deprecated. Change the call to include dim=X as an argument.
  out = F.softmax(self.fc_out(x))#,dim=1) #  UserWarning: Implicit dimension choice for
For the time being set dim to 1
